onnxruntime/test/onnx/TestCase.cc

```diff
@@ -203,7 +204,10 @@ class OnnxModelInfo : public TestModelInfo {
     if (!model_pb.ParseFromZeroCopyStream(&f)) {
       ORT_THROW("Failed to load model because protobuf parsing failed.");
     }
-
+#ifdef __GNUG__ 
+    std::string url_string{model_url};
+    onnx_commit_id_ = url_string.size() > 50 ? url_string.substr(11,40) : "";   
```

Comment:
👎

Reply:
Added logic handling wchar_t.
Yes it is ugly still.